tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -28,7 +28,7 @@ STATIC_ASSERT(__cpp_lib_adaptor_iterator_pair_constructor == 202106L);
 STATIC_ASSERT(__cpp_lib_addressof_constexpr == 201603L);
 #endif
 
-#if _HAS_CXX20 && defined(__cpp_lib_concepts) // TRANSITION, GH-395
+#ifdef __cpp_lib_concepts

```

Comment:
What's up with this change?

Reply:
This was copied from other additions. I think `defined(__cpp_lib_concepts)` subsumes `_HAS_CXX20`.

Reverting now as unneeded.

Reply:
Yeah, I just don't like the drive-by here.